ch_pipeline/processing/daily.py

```diff
@@ -241,8 +242,29 @@
         weight: natural
         exclude_intracyl: true
         include_auto: false
+        
+    # Truncate ringmaps before writing to disk
+    - type: draco.core.io.Truncate
+      requires: manager
+      in: ringmap
+      out: ringmap_truncated
+      params:
         save: true
-        output_name: "ringmap_intercyl_{{tag}}.h5"
+        output_name: "ringmap_{{tag}}.zarr"
+        output_format: zarr
+        output_compression: "bitshuffle"
+        output_compression_opts: [0, "lz4"]
+    - type: draco.core.io.Truncate
```

Comment:
Tristan made changes to `Truncate`, so we should check to make sure this config still holds.